src/in-depth/human-communication.md

```diff
@@ -2,17 +2,22 @@
 
 ## When everything is fine
 
-<aside class="todo">
-
-**TODO:**
-Style of writing:
-- informative and concise
-- easy to parse
-- consistent: use same prefixes, sentence structure
-
-[Issue #74](https://github.com/rust-lang-nursery/cli-wg/issues/74)
-
-</aside>
+When communicating progress while your application is doing
+fine it's important to be informative and consise in log messages.
```

Comment:
Hm, this seems like a hard to parse sentence to me. But it is also Monday morning. Maybe rearrange it a bit by starting with

> It is useful to report on the application's progress even when everything is fine.

And then dropping the important bit in a new sentence

> Try to be informative and concise in these messages.

(The rest is great!)